packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config-table.vue

```diff
@@ -165,26 +181,80 @@ const openMatrixModal = (datum: ModelConfigTableData) => {
 
 const rowClass = (rowData) => (rowData.type === ParamType.MATRIX ? '' : 'no-expander');
 
+const updateTimeseries = (id: string, value: string) => {
+	if (!validateTimeSeries(value)) return;
+	const clonedConfig = cloneDeep(props.modelConfiguration);
+	clonedConfig.configuration.metadata.timeseries[id] = value;
+	emit('update-configuration', clonedConfig);
+};
+
+const validateTimeSeries = (values: string) => {
+	let isValid = true;
+	if (typeof values !== 'string') {
+		isValid = false;
+		errorMessage.value = 'Incorrect Format (e.g., 0:500, 10:550, 25:700 etc)';
+		return isValid;
+	}
+	const timeValuePairs = values.split(',');
+
+	timeValuePairs.forEach((pair) => {
+		const colonCount = pair.split(':').length - 1;
+
+		// If a pair does not contain exactly one colon, return false
+		if (colonCount !== 1) {
+			isValid = false;
+			return;
+		}
+
+		const [time, value] = pair.split(':').map((part) => part.trim());
+
+		if (!/^\d+$/.test(time) || !/^\d+(\.\d+)?$/.test(value)) {
+			isValid = false;
+		}
+	});
```

Comment:
FYI: This forEach is a bit inefficient, you will keep on parsing even if isValid is false

Reply:
changed in favour for the function yohann provided